setup.py

```diff
@@ -3,7 +3,7 @@
 from setuptools import setup
 from sys import version_info
 
-__version = "3.9.2"
+__version = "3.9.3"
```

Comment:
Please increase version. I see a number of commits since last successful deploy.

Reply:
Done, 3.9.4